Logger/Logger.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+#include "MutexQueue.h"
+#include "Message.h"
+#include "Utils.h"
+#include <thread>
+#include <iostream>
+#include <atomic>
+
+namespace SecureBrowser
+{
+
+enum class LogLevel
+{
+	Debug,
+	Info,
+	Warning,
+	Error,
+};
+
+struct LogMessage
```

Comment:
You can hide this struct since there are no available API to operate with the struct